Agreements/External/DUBN_IoM.txt

```diff
@@ -0,0 +1 @@
+FIR_COPX:EIDW:*:GINIS:*:*:DUBN:Isle of Man:23000:*:GINIS
```

Comment:
```suggestion
FIR_COPX:EIDW:*:GINIS:*:*:DUBN:Isle of Man:23000:*:^GINIS
```

Not got round to agreements from external sectors yet, but may as well change this one now!